utils/actorCode/index.ts

```diff
@@ -0,0 +1,14 @@
+import base32Decode from 'base32-decode'
+
+export function decodeActorCID(cid: string): string {
+  try {
+    const decoded = base32Decode(cid.slice(1).toUpperCase(), 'RFC4648')
+    const actorCode = new TextDecoder('utf-8').decode(decoded.slice(4))
+    if (!actorCode.includes('fil/')) {
+      throw new Error('unknown actor code')
+    }
```

Comment:
@Schwartz10 key part to document here is that when "decoding" the typical cid of:
`{ver varint}` `{codec varint}` `{mh varint}` `{len varint}` `{payload}`
we simply assume that the first 4 varints are 1 byte each, do not check any of them and just validate whether the payload is a valid string